production/helm/loki/templates/_helpers.tpl

```diff
@@ -308,12 +312,15 @@ filesystem:
 {{- end -}}
 {{- end -}}
 {{- end -}}
+{{- end -}}
 
 {{/*
 Storage config for ruler
 */}}
 {{- define "loki.rulerStorageConfig" -}}
-{{- if .Values.minio.enabled -}}
+{{- if .Values.loki.storage.use_thanos_objstore -}}
```

Comment:
this is not needed here right? since we only need to add thanos specific ones under ruler_storage block

Reply:
good catch! 👍🏽